delfin/common/constants.py

```diff
@@ -60,3 +60,49 @@ class VolumeType(object):
     THIN = 'thin'
 
     ALL = (THICK, THIN)
+
+
+class PortType(object):
+    FRONT_END = 'front_end'
```

Comment:
Please put a comment for the type of port, ex: FRONT_END = Masking port BACK_END = Mapping port or device port etc..

Reply:
done, thanks